.github/workflows/manual-deploy-testnet-l2.yml

```diff
@@ -358,7 +360,7 @@ jobs:
         run: |
           curl -XPOST -H "Authorization: Bearer ${{ secrets.GH_TOKEN }}" -H "Accept:application/vnd.github" -H "Content-Type:application/json" https://api.github.com/repos/obscuronet/faucet/actions/workflows/manual-deploy-${{ github.event.inputs.testnet_type }}-faucet.yml/dispatches --data '{"ref": "main" }'
 
-      - name: 'Trigger Faucet deployment on repository_dispatch'
-        if: ${{ github.event_name == 'repository_dispatch' }}
+      - name: 'Trigger Faucet deployment on schedule'
+        if: ${{ github.event_name == 'schedule' }}
```

Comment:
Confused why this condition exists (not that it's new in this PR). Does this stop it running on main testnet?

Reply:
It helps differentiate the faucet deployment from one where it was done manually e.g. through a new release, and one where it is done through a schedule task ... the faucet is responsible for triggering the E2E tests and we only want to do this on the nightly schedule, not on a manual deployment (of testnet or dev-testnet)